c7n/resources/dynamodb.py

```diff
@@ -240,6 +240,138 @@ def process(self, resources):
                                 f.exception()))
 
 
+@Table.action_registry.register('create-backup')
+class CreateBackup(BaseAction, StatusFilter):
+    """Creates a manual backup of a DynamoDB table
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: dynamodb-create-backup
+                resource: dynamodb-table
+                actions:
+                  - type: create-backup
+    """
+
+    valid_status = ('ACTIVE',)
+    schema = type_schema('create-backup')
+    permissions = ('dynamodb:CreateBackup',)
+
+    def process(self, tables):
+        tables = self.filter_table_state(
+            tables, self.valid_status)
+        if not len(tables):
+            return
+
+        with self.executor_factory(max_workers=3) as w:
+            futures = []
+            for t in tables:
+                futures.append(w.submit(
+                    self.process_dynamodb_tables,
+                    tables))
```

Comment:
oops... your taking quite a few backups here.. for each single table your backing up the full set, I'd suggest using chunks utility function to get an iterator over subsets.